angr/analyses/stack_pointer_tracker.py

```diff
@@ -63,61 +250,61 @@ def __init__(self, func : Function):
             func = func.copy()
             func.normalize()
 
+        self.track_mem = track_memory
         self._func = func
-        self._states = { }
-        self._insn_to_sp_offset = defaultdict(dict)
-        self._insn_to_bp_offset = defaultdict(dict)
-        self.sp_inconsistent = False
+        self.regOffsets = regOffsets
```

Comment:
Can we use `reg_offsets` instead? `regOffsets` is unusual for naming class members.